TFUtil.py

```diff
@@ -2688,24 +2706,33 @@ def reversed(x):
   return y
 
 
-def flatten_with_seq_len_mask(x, seq_lens, time_major=False):
+def flatten_with_seq_len_mask(x, seq_lens, batch_dim_axis=0, time_dim_axis=1, time_major=False):
```

Comment:
You added some conflicting params (`time_dim_axis`, `batch_dim_axis` are conflicting with `time_major`). You cannot have the default params like that. Keep the default params all as `None`, extend the documentation about the default param, and add checks in the code, that only either `time_major` or `batch_dim_axis` and `time_dim_axis` can be set.

Reply:
Ok